WordPress/Classes/ViewRelated/Reader/ReaderSeenAction.swift

```diff
@@ -0,0 +1,7 @@
+/// Encapsulates a command to toggle a post's seen status
+final class ReaderSeenAction {
+    func execute(with post: ReaderPost, context: NSManagedObjectContext, completion: (() -> Void)? = nil) {
```

Comment:
This method accepts a completion block, that can be nil, but then the block is never called?

Reply:
🤦 You know when you look at a dumb mistake and you flash back to exactly the moment you made it and know exactly what you did? Yea, that.